projects.json

```diff
@@ -16,10 +17,11 @@
     },
     {
       "id": 2,
-      "name": "Création du site 'csesmi.fr' pour le CSE d'Accor SMI",
+      "name": "Creation of the 'csesmi.fr' website for the Social and Economic Comitee of Accor SMI",
       "mainImg":"/media/wordpress.svg",
       "link": "https://www.csesmi.fr/",
-      "description":"Lors de mon premier stage de deux mois, j'avais pour mission de créer le site du CSE d'Accor SMI, à l'aide de WordPress, de beaucoup de CSS et de Woocommerce. De plus, j'avais aussi pour mission de former leurs équipes à l'utilisation du-dit site (Gestion d'articles, support à faire, gérer les commandes). <+Enfin, j'assure désormais la maintenance, le support et la vie du site en freelance pour Accor SMI].",
+      "description":"During my first 2 month internship, i had to create the website for the Social and Economic Comitee of Accor SM, using Wordpress, a lot of CSS and Woocommerce. Morevover, i had to teach their teams how to use the created website(Articles management, commands). <+ Lastly, i now provide the maintenance, the IT support and the updates on the life of the website as a freelancer for Accor SMI].",
```

Comment:
I had (capitalization of the I)
Moreover I had to teach their staff how to use the website, such as handling the orders and how to manage the articles and comments.
<+ to change into a <-
Lastly, i now provide the maintenance, the IT support and the updates on the life of the website as a freelancer for Accor SMI -> Lastly, I am now in charge of the maintenance, IT Support and updating of the website as a freelancer.

Reply:
the website's functionalities